sim_learn.py

```diff
@@ -1,11 +1,67 @@
+"""
+This module is a command line tool which provides an interface for experiments which are designed to learn an arbiter
+PUF LTFarray simulation with the logistic regression learning algorithm. If you want to use this tool you will have to
+define nine parameters which define the experiment.
+
+Example usage "python sim_learn.py n k transformation combiner N restarts instances seed_instance seed_model"
```

Comment:
do not include function signature here, as we already have two places to document this (stdout below, and readme)